src/main/java/org/reaktivity/nukleus/tcp/internal/stream/Acceptor.java

```diff
@@ -65,7 +65,7 @@
 
     public Acceptor()
     {
-        this.sourcesByLocalAddress = new HashMap<>();
+        this.sourcesByLocalAddress = new TreeMap<>((a, b) -> addressesMatch(a, b) ? 0 : 1);
         this.registerHandler = this::handleRegister;
```

Comment:
Recommend converting `addressesMatch` to return an `int` such that it can be passed directly as a static lambda.

Reply:
done